src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  justify-content: center;
+
+  width: 200px;
+  min-height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  box-sizing: border-box;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  width: 100%;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+
+  padding-top: 40px;
+  padding-left: 16px;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  flex-basis: 100%;
+  line-height: 14px;
+  color: #616070;
+
+  padding-left: 16px;
+  padding-top: 4px;
+}
+
+.star-box__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+}
```

Comment:
You need to align the 2 parts of your .star-box - the stars and the text. Good practice will be to use flexbox here. I'll give you a hint: you need to justify content so one part stick to the left and the other to the right. And you need to center these parts